apps/docs/src/app/core/component-docs/file-uploader/examples/file-uploader-example/file-uploader-example.component.html

```diff
@@ -0,0 +1,21 @@
+<fd-file-uploader
+    #fileInput
+    [id]="'fileInput'"
+    [ariaLabel]="'Choose file'"
+    [ariaLabelledBy]="'Choose file'"
+    [placeholder]="'Select the file '"
+    [buttonLabel]="'Browse'"
```

Comment:
in one of my PR, got comments to use like id="fileInput", in case of string and boolean. same can be followed.

Reply:
modified.